pycloudlib/lxd/instance.py

```diff
@@ -190,17 +216,23 @@ def restore(self, snapshot_name):
                         self.name, snapshot_name)
         subp(['lxc', 'restore', self.name, snapshot_name])
 
-    def shutdown(self, wait=True):
+    def shutdown(self, wait=True, force=True):
```

Comment:
We should change pycloudlib/instance.py and pycloudlib/*/instance.py:shutdown and restart methods to accept **kwargs
```suggestion
    def shutdown(self, wait=True, force=True, **kwargs):
```